tsl/src/bgw_policy/continuous_aggregate_api.c

```diff
@@ -110,21 +110,21 @@ get_time_from_config(const Dimension *dim, const Jsonb *config, const char *json
 }
 
 int64
-policy_refresh_cagg_get_refresh_start(const Dimension *dim, const Jsonb *config)
+policy_refresh_cagg_get_refresh_start(const Dimension *dim, const Jsonb *config, bool *start_isnull)
 {
-	bool start_isnull;
-	int64 res = get_time_from_config(dim, config, POL_REFRESH_CONF_KEY_START_OFFSET, &start_isnull);
+	int64 res = get_time_from_config(dim, config, POL_REFRESH_CONF_KEY_START_OFFSET, start_isnull);
+
 	/* interpret NULL as min value for that type */
 	if (start_isnull)
```

Comment:
Shouldnt this dereference the pointer now since you change the type
```suggestion
	if (*start_isnull)
```